src/app/component/ProductListPage/ProductListPage.component.js

```diff
@@ -49,6 +49,20 @@ export class ProductListPage extends PureComponent {
         mix: {}
     };
 
+    state = {
+        siblingsHaveBrands: false,
+        siblingsHavePriceBadge: false,
+        siblingsHaveTierPrice: false,
+        siblingsHaveConfigurableOptions: false
+    };
```

Comment:
Same here as above

Reply:
We need to use parent component state in order to notify other product cards that at least one of them has these elements rendered.
Then other cards will reserve an empty space in order to line up other elements in the layout.